disnake/flags.py

```diff
@@ -1598,6 +1598,32 @@ def automod(self):
         """
         return (1 << 20) | (1 << 21)
 
+    @flag_value
+    def guild_polls(self):
```

Comment:
Could you add a `polls` alias to set both intent flags in one go, similar to `typing`, and mention that intent in the event descriptions?